about.html

```diff
@@ -20,45 +23,49 @@
     </div>
   </header>
   <h1>About</h1>
-  <h3>Room: 307</h3>
-  <h3>Time: Mondays - Thursdays, immediately after 10th period - 5:00pm</h3>
-  <h2>What is the dojo?</h2>
+  <h4>Room: 307</h4>
+  <h4>Time: Mondays - Thursdays, immediately after 10th period - 5:00pm</h4>
+  <h3>What is the dojo?</h3>
   <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
     tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
     quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
     consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
     cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
     non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-  <h2>Rules</h2>
-  <ol start = 0>
-    <li>All lists must be 0-indexed.</li>
-    <li>No food or drink in the dojo (H20 okay).</li>
-    <li>Be respectful to your peers.</li>
-    <li>Clean up after yourself.</li>
-    <li>Don't make excessively loud noises.</li>
-    <li>Avoid using illegal internet (Facebook/Messenger).</li>
-    <li>Keep the aisles clear of backpacks.</li>
-    <li>Lock your workstation when not in use (CTRL + ALT + L).</li>
+<h3>Rules</h3>
+
+  <ol start = 0 class="mdl-list">
+    <li class="mdl-list__item">All lists must be 0-indexed.</li>
```

Comment:
Please have numbers:
0. Rule 0
1. Rule 1
2. Rule 2

Here is how to do so: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_ol_start